xds/src/main/java/io/grpc/xds/internal/rbac/engine/EvaluateArgs.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal.rbac.engine;
+
+import io.grpc.Grpc;
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+
+/** The EvaluateArgs class holds evaluate arguments used in CEL Evaluation Engine. */
+public class EvaluateArgs {
+  private Metadata headers;
+  private ServerCall<?, ?> call;
+
+  /**
+   * Creates a new evaluate argument using the input {@code headers} for resolving headers
+   * and {@code call} for resolving gRPC call.
+   */
+  public EvaluateArgs(Metadata headers, ServerCall<?, ?> call) {
+    this.headers = headers;
+    this.call = call;
+  }
+
+  /** Return the headers. */
+  public Metadata getHeaders() {
+    return headers;
+  }
+
+  /** Return the gRPC call. */
+  public ServerCall<?, ?> getCall() {
+    return call;
+  }
+
+  /** Extract the request.url_path field. */
+  public String getRequestUrlPath() {
+    String requestUrlPath = this.getCall().getMethodDescriptor().getFullMethodName();
```

Comment:
Any reason for using `getCall` instead of directly using this.call? Mocking `getCall` could be the only reason but why would you mock a simple getter?

Reply:
Fixed!